web/source/kmwnative.ts

```diff
@@ -80,7 +82,7 @@ if(!window['keyman']['initialized']) {
      * @param {string}  Lfilename  keyboard file name with optional prefix                     
      */   
     keymanweb.getKeyboardPath=function(Lfilename) {           
-      var rx=RegExp('^(([\.]/)|([\.][\.]/)|(/))|(:)');   
+      var rx=RegExp('^(([\\.]/)|([\\.][\\.]/)|(/))|(:)');   
```

Comment:
Why aren't we using `var rx=/..../` instead? Avoids escapes. See also all over the place.

Reply:
That's a good question.  I usually go with 'don't change it' when I run into spots like this, so I left it with the old format.  I'll go ahead and convert it where I see it.

Reply:
That being stated, in this particular case, please note that there are multiple internal forward slashes.  It's literally impossible to avoid escapes for this particular example - the question is if we prefer the escape `\.` over the escape `\/`.

Reply:
Gotcha. Don't change the regexes in this PR -- we can do them later in a separate PR if we decide it is important.